recipes/metamorpheus/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set version = "0.0.316" %}
+{% set sha256 = "389c7446abe61a931a78f3c27478baadf6cb43d9a00840b954c22a3ec429a316" %}
+
+package:
+  name: metamorpheus
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/smith-chem-wisc/MetaMorpheus/releases/download/{{ version }}/MetaMorpheus_CommandLine.zip
+  sha256: '{{ sha256 }}'
```

As it's only used once I find it's nice to include directly rather than using a variable:
```suggestion
  sha256: 389c7446abe61a931a78f3c27478baadf6cb43d9a00840b954c22a3ec429a316
```